WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivityVideoHelper.java

```diff
@@ -155,6 +227,9 @@ public void onError(Exception exception) {
             if (isStopped) {
                 return;
             }
+
+            trackVideoProcessingEvents(true, exception);
```

Comment:
Yes, I had though not to track the exception in case of errors. Will change it, since it's better to track them even when `isStopped` is true.